pipeline_tools/create_envelope.py

```diff
@@ -39,7 +40,10 @@ def run(submit_url, analysis_json_path, analysis_file_version):
     with open(analysis_json_path) as f:
         analysis_json_contents = json.load(f)
     analyses_url = get_subject_url(envelope_js, 'processes')
-    analysis_js = create_analysis(analyses_url, auth_headers, analysis_json_contents, http_requests)
+
+    analysis_js = get_analysis_results(analyses_url, auth_headers, analysis_json_contents, http_requests)
```

Comment:
Worth a comment explaining that we may have already created the analysis process in a previous attempt and that's why we're checking for it before creating it. Maybe rename the function get_analysis_process?